src/dbus_server.rs

```diff
@@ -8,21 +8,36 @@ use zbus::dbus_interface;
 struct Server {
     tx: mpsc::Sender<Request>,
     color: Color,
+    output_name: Option<String>,
 }
 
-pub async fn run(tx: mpsc::Sender<Request>) -> Result<bool> {
-    match zbus::ConnectionBuilder::session()?
-        .serve_at(
+pub async fn run(tx: mpsc::Sender<Request>, output_names: Vec<String>) -> Result<bool> {
+    let mut connection_builder = zbus::ConnectionBuilder::session()?;
+
+    let no_output_names = output_names.is_empty();
+    for output_name in output_names {
+        connection_builder = connection_builder.serve_at(
+            format!("/Output/{}", output_name.replace('-', "_")),
```

Comment:
As far as I'm aware there is no strict naming convention here: `upower` has `/devices/` while `networkmanager` has `/Devices/`, but maybe let's use `/outputs/`?